### ISSUE TITLE
Move the score title tab

### ISSUE THREAD

**Issue by runarcn** [musescore/MuseScore#22425]
### Your idea

Currenlty, MuseScore 4 and upwards is unable to have two scores open at the same time. However, MS4 still has a tab view for currently open scores. This is superfluous, takes up screen space, and gives an impression that you can open two scores in the same window. (pictures below)

### Problem to be solved

I suggest that we remove the tab-view until this is implemented, and move the score title to the center top of the window. I've added a short mockup I made of how this could look in GIMP further down.

### Prior art

The bar in question:
![image](https://github.com/musescore/MuseScore/assets/52424029/5a4f3854-f83e-44e2-b926-8769c3b0f161)

How I suggest this be made:
![image](https://github.com/musescore/MuseScore/assets/52424029/b17d2746-2a27-453f-bfab-c40132100fbd)

**Comment by runarcn** [comment]
It seems that it is there by default if you use a normal desktop instead of a minimal Linux window manager :P 
However, the option to hide/remove/move the score title in the tab view would still be a nice QoL addition.
![image](https://github.com/musescore/MuseScore/assets/52424029/9cc21e32-ac37-46d4-92ab-bbafa9d949d9)

**Comment by Fristover** [comment]
The tab system is still used for the parts, though, that's why it's still there. 😅

**Comment by cbjeukendrup** [comment]
Exactly, this tab bar has the purpose of showing part scores. One could argue that we could as well hide it whenever there is only one tab, but for the rest nothing about it is not implemented.

**Comment by cbjeukendrup** [comment]
Actually the fact that the score name isn't already in the window title is a bit weird, and a bug. On macOS and Windows, it is there.

**Comment by runarcn** [comment]
> The tab system is still used for the parts, though, that's why it's still there. 😅
> 
> Like this: ![image](https://private-user-images.githubusercontent.com/68985619/322941572-f6e4e423-5e87-41fc-914c-74b6de8ccef7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzMzU3NDYsIm5iZiI6MTcxMzMzNTQ0NiwicGF0aCI6Ii82ODk4NTYxOS8zMjI5NDE1NzItZjZlNGU0MjMtNWU4Ny00MWZjLTkxNGMtNzRiNmRlOGNjZWY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE3VDA2MzA0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBjMGQwZDQzYzczZmVhOTBiZTM1ZDI0NTNhNzMzY2JjNjAyZTIxNzMxZmIzMzRmMDExMDcyZTFmZTkzOGQxOTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.aKbZedMS3O6uI_8H7N7-3eDxu5qwq958CPEAlKjU9NY)

:P my mistake, think I brainfarted and thought it was still like on MS3 with parts appearing below 

> Exactly, this tab bar has the purpose of showing part scores. One could argue that we could as well hide it whenever there is only one tab, but for the rest nothing about it is not implemented.

That would be very cool!
